src/routes/(app)/+layout.svelte

```diff
@@ -555,7 +564,14 @@
 						<Picture class={logoClass} picture={data.footerPicture} />
 					</div>
 				{:else if data.displayPoweredBy}
-					<div class="flex w-full">
+					<div class="w-full hidden sm:flex">
+						<a class="items-center gap-4" href="https://be-bop.io" target="_blank">
+							<span class="font-light">{t('footer.poweredBy')} </span>
+							<img class="h-[40px] w-auto hidden dark:inline" src={DEFAULT_LOGO} alt="" />
+							<img class="h-[40px] w-auto dark:hidden" src={DEFAULT_LOGO_DARK} alt="" />
+						</a>
+					</div>
+					<div class="sm:hidden justify-center flex w-full">
```

Comment:
```suggestion
					<div class="justify-center sm:justify-normal flex w-full">
```

no?

and then no need for copy paste